_videos/2017-01-30-complex-refactoring-in-simple-steps-III.md

```diff
@@ -0,0 +1,21 @@
+---
+author: Matthew Butt
+layout: video
+name: complex-refactoring-in-simple-steps-III
+image:
+ src: /assets/img/custom/videos/complex-refactoring-in-simple-steps-III.jpg
+
+title: "Complex refactoring in simple steps Part III"
+hidden: true
+video-url: https://www.youtube.com/embed/SuqrAtXnn1U
+date: 2017-01-30 10:00:00 +00:00
+---
+
+
+## Screencast - Complex refactoring in simple steps, Part III From Conditional to Polymorphism
+
+Matthew Butt demonstrates how to compose simple automated refactoring steps in ReSharper to refactor code to better designs.
```

Comment:
Your phrasing is certainly more elegant, avoiding the repetition of 'to', but doesn't quite capture the nuance I wanted.